test/sentry/live_view_hook_test.exs

```diff
@@ -31,11 +31,44 @@ defmodule SentryTest.LiveComponent do
   end
 end
 
+defmodule SentryTest.DeadLive do
+  use Phoenix.LiveView
+  on_mount Sentry.LiveViewHook
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div>I'm being live_rendered!</div>
+    """
+  end
+end
+
+defmodule SentryTest.ErrorView do
+  def render(template, _), do: Phoenix.Controller.status_message_from_template(template)
+end
```

Comment:
Do you need this considering you're just returning some text with `text/2`?

Reply:
Nope! Removed!